tests/annotation_model_tests/test_settings.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+model_path_sv = os.path.abspath(
+    "./saved_models/Superview_Nieuwkoopse_plassen_20190302_113613_to_20221012_104900_random_forest_classifier.sav"
+)
+model_path_pneo = os.path.abspath(
+    "./saved_models/PNEO_Nieuwkoopse_plassen_20230603_to_20230905_random_forest_classifier.sav"
```

Comment:
How would someone obtain these saved_models? Maybe add them in the instruction in ./saved_models/README.md?

Reply:
download from the blob storage, for now only a upload functions to the train model notebook.

Reply:
fixed by using small local models files for now